src/Provider.js

```diff
@@ -7,6 +7,7 @@ export const MobXProviderContext = createContext({})
 
 export class Provider extends Component {
     static contextType = MobXProviderContext
+    static displayName = "Provider"
```

Comment:
Perhaps `MobXProvider` might be a more useful name to distinguish from other providers?

Reply:
I thought about it, but decided to stay at the more conservative decision. If you think that this needs to be changed, then I will do it.

Reply:
Please do.

Reply:
Done.